content/en/docs/components/pipelines/v2/compile-a-pipeline.md

```diff
@@ -1,104 +1,75 @@
 +++
-title = "Compile a Pipeline"
+title = "Compile a pipeline"
 description = "Compile a pipeline definition to YAML"
 weight = 5
 +++
 
-While pipelines and components are authored in Python, they can be compiled to intermediate representation (IR) YAML ([example][compiled-output-example]).
+You can compile your pipeline or component to intermediate representation (IR) YAML.  The IR YAML definition preserves a static representation of the pipeline or component. You can submit the YAML definition to the KFP backend for execution, or deserialize it using the KFP SDK for integration into another pipeline.
+([View an example on Github][compiled-output-example]).
 
-A YAML pipeline/component definition preserves a static representation of your pipeline/component. This YAML can be submitted to the KFP backend for execution or deserialized by the KFP SDK for integration into another pipeline.
+**Note:** Pipelines as well as components are authored in Python. A pipeline is a template representing a multistep workflow, whereas a component is a template representing a single step workflow.
 
-## Compiling
-First let's define a very simple pipeline:
+## Compile a pipeline
 
-```python
-from kfp import compiler
-from kfp import dsl
+Pipelines as well as components are workflow templates authored in Python. A pipeline represents a multistep workflow, whereas a component represents a single step workflow. You can compile a pipeline or a template into IR YAML using the `Compiler.compile` method. To do this, follow these steps:
```

Comment:
This is duplicative of the `Note:` above

Reply:
Fixed